docs/spec.md

```diff
@@ -2823,6 +2825,7 @@ More formally, `result[result_index] = operand[operand_index]` where:
 * `operand_index` = `add(full_start_index, full_offset_index)`.
   If `operand_index` is out of bounds for `operand`, then the behavior is
   implementation-defined.
```

Comment:
"If operand_index is out of bounds for operand, then the behavior is implementation-defined". Does this still apply?

Reply:
Thanks, this should be removed.